Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -1310,6 +1310,14 @@ between them. -/
 def toRightMovesNeg {x : PGame} : x.LeftMoves ≃ (-x).RightMoves :=
   Equiv.cast (rightMoves_neg x).symm
 
+/-- Converts x.RightMoves to (-x).LeftMoves, as proved in `toLeftMovesNeg`. -/
+def asLeftMovesNeg {x : PGame} (i : x.RightMoves) : (-x).LeftMoves :=
+  (leftMoves_neg x).mpr i
```

Comment:
As the `leftMoves_neg` docstring suggests, you should instead use [`SetTheory.PGame.toLeftMovesNeg`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Game/PGame.html#SetTheory.PGame.toLeftMovesNeg) to cast between both types.

Reply:
I remember trying `toLeftMovesNeg` before and encountering an issue, but it works fine with `toLeftMovesNeg`. Fixed :+1: